tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -35,4 +37,9 @@ TEST_F(WholeFileTest, ConstantPixelShader) {
                    /*generateHeader*/ true,
                    /*runValidation*/ true);
 }
+
+TEST_F(FileTest, CheckMemoryModelAndEntryPoint) {
+  runFileTest("check-entrypoint.hlsl2spv",
```

Comment:
I'd suggest to leave `.hlsl2spv` suffix only for whole file check tests and use `.hlsl` for FileCheck style tests. So it's clear in the directory.